compiler/rustc_middle/src/hir/intravisit.rs

```diff
@@ -0,0 +1,28 @@
+pub mod nested_filter {
```

Comment:
Why a nested module? Can't the file be directly called `nested_filter.rs`?

Reply:
So it is `hir::nested_filter` instead of `hir::intravisit::nested_filter`? Yeah that would be fine.